CMSgov/mint-app
# [EASI-3413] - Email template standardization

# EASI-3413

## Changes and Description

- Centered all emails within `<table />` elements
- Standardized styles of all email templates into `shared_style.html`
- Extracted header of all templates into `shared_header.html`
- Extracted footer of all templated into `shared_footer.html`
- Extracted access banner into `shared_access_banner.html`



## How to test this change

Mailcatcher - [http://127.0.0.1:1085/](http://127.0.0.1:1085/)

- For all emails outside of cron scheduled jobs
  - Run `scripts/dev email`
  - Verify emails render correctly

- For cron scheduled emails
  - Seed db
  - Run `UPDATE audit.change SET modified_dts = modified_dts - INTERVAL '1 day';` in Postico/sql tooling to set changes to yesterday
  - Optional (set the cron schedule to 1 min - `schedule = "* * * * *"`
  - Open Faktory in browser to monitor schedule - [http://localhost:7420/loops](http://localhost:7420/loops)
  - Verify emails render correctly

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
